ts/utils/isVersionAppSupported.ts

```diff
@@ -0,0 +1,14 @@
+import { ServerInfo } from "../../definitions/backend/ServerInfo";
```

Comment:
instead of **isVersionAppSupported.ts** (and the test file too)  you could name this file **appVersion.ts**
**isVersionAppSupported** is referring to a function instead a set of functions regarding version (maybe in future we need mode utils functions to add about version)